framework/filters/PageCache.php

```diff
@@ -123,6 +124,13 @@ class PageCache extends ActionFilter
      * @since 2.0.4
      */
     public $cacheHeaders = true;
+    /**
+     * @var array a list of placeholders for embedding dynamic contents. This property
+     * is used internally to implement the content caching feature. Do not modify it.
+     * @internal
```

Comment:
a public $variable which we don't want to modify? Shouldnt we use a setter than?